### PR TITLE
Azure keyvault trust manager with multiple keystores.

### PR DESCRIPTION
**Pull request by michaelqi793**
Currently, the Azure keyvault trust manger employs either the keyvault keystore or the local JRE keystore. Now it comploys both.

### CODE REVIEW

**Review by yiliuTo — COMMENTED** [review]
LGTM, could you add necessary unit tests for the constructor?

**Comment by backwind1233** [comment]
Here multiple means KeyVaultTrustManager has two TrustManager fields, one for keyvault, another is for default keyStore.

**Review by yiliuTo — APPROVED** [review]
LGTM.

File: sdk/keyvault/azure-security-keyvault-jca/pom.xml [hunk 1]
```diff
@@ -205,6 +205,6 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jacoco.min.branchcoverage>0</jacoco.min.branchcoverage>
-        <jacoco.min.linecoverage>0.05</jacoco.min.linecoverage>
+        <jacoco.min.linecoverage>0</jacoco.min.linecoverage>
```
Comment (backwind1233) [inline sdk/keyvault/azure-security-keyvault-jca/pom.xml:208]: why you change the value to 0

**Comment by chenrujun** [comment]
Hi, @michaelqi793 , could you please fix the pipeline failure?

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java [hunk 2]
```diff
@@ -37,31 +43,71 @@
      * @param keyStore the keystore.
      */
     public KeyVaultTrustManager(KeyStore keyStore) {
-        this.keyStore = keyStore;
-        if (this.keyStore == null) {
-            try {
-                this.keyStore = KeyStore.getInstance(KeyVaultKeyStore.KEY_STORE_TYPE);
-                this.keyStore.load(null, null);
-            } catch (KeyStoreException | IOException | NoSuchAlgorithmException | CertificateException ex) {
-                ex.printStackTrace();
```
Comment (stliu) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java:46]: Never, never have `printStackTrace()` in our production code.

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java [hunk 3]
```diff
@@ -37,31 +43,71 @@
...
+        addDefaultTrustManager();
+
+    }
+
+    private void addKeyVaultKeystore() {
+        try {
+            this.keyStore = KeyStore.getInstance(KeyVaultKeyStore.KEY_STORE_TYPE);
+            this.keyStore.load(null, null);
+        } catch (KeyStoreException | IOException | NoSuchAlgorithmException | CertificateException ex) {
+            ex.printStackTrace();
```
Comment (stliu) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java:77]: see above comment, use log!

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java (hunk 2 above)
Comment (michaelqi793) [reply sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java:46]: Replaced

**Pull request merged**